apps/stats-web/.env.local.sample

```
@@ -1,11 +1,11 @@
 
-NEXT_PUBLIC_API_BASE_URL=http://localhost:3080
-NEXT_PUBLIC_NODE_ENV=$NODE_ENV
-NEXT_PUBLIC_BASE_API_MAINNET_URL=$NEXT_PUBLIC_API_BASE_URL
-NEXT_PUBLIC_BASE_API_TESTNET_URL=$NEXT_PUBLIC_API_BASE_URL
-NEXT_PUBLIC_BASE_API_SANDBOX_URL=$NEXT_PUBLIC_API_BASE_URL
-NEXT_PUBLIC_DEFAULT_NETWORK_ID=mainnet
+VITE_API_BASE_URL=http://localhost:3080
+VITE_NODE_ENV=$NODE_ENV
+VITE_BASE_API_MAINNET_URL=$VITE_API_BASE_URL
+VITE_BASE_API_TESTNET_URL=$VITE_API_BASE_URL
+VITE_BASE_API_SANDBOX_URL=$VITE_API_BASE_URL
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_BASE_API_SANDBOX_URL key should go before the VITE_BASE_API_TESTNET_URL key

(UnorderedKey)